client/src/components/documents/Document.tsx

```diff
@@ -15,13 +15,13 @@ import {
 import React, { useState, useEffect, useMemo } from 'react';
 import { useParams, useNavigate } from 'react-router-dom';
 import { DocumentPageMap, PreviewMap } from '../map/Map';
-import { KxDocument, DocCoords } from "../../model";
+import { KxDocument, DocCoords, Scale, ScaleOneToN } from "../../model";
 import { mongoose } from '@typegoose/typegoose';
 import "../../css/document.css";
 import PreviewDoc from './Preview';
 import { Toast } from '@radix-ui/react-toast';
 import { Toaster } from '../toast/Toaster';
-import { AreaType, KxDocumentType, Scale, Stakeholders } from "../../enum";
+import { AreaType, KxDocumentType, ScaleType, Stakeholders } from "../../enum";
```

Comment:
<img width="1440" alt="Screen Shot 2024-12-11 at 10 23 16" src="https://github.com/user-attachments/assets/73b654b2-2bbf-4f29-a4a4-36d108c08230">
by clicking a document on the map, and redirecting to the details, the page becomes blank.